### PR TITLE
swagger.yaml edits for liquidity module

### PR DESCRIPTION
**Pull request by barriebyron**
## Description

swagger.yaml edits for liquidity module

question: do we want a message description for block heights to be "is" or "was"
in this PR, I am suggesting to use present tense (although I was tempted to use the past tense, I don't know enough yet to make a firm recommendation)

I think that we are recording the block height "when this message _was_ appended" ?? 

> title: block height when this message is appended to the batch

closes: part of https://github.com/tendermint/liquidity/pull/217
---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by kogisin — COMMENTED** [review]
LGTM other than the question I commented if we should end sentence with a dot for the title section.

File: client/docs/swagger-ui/swagger.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ paths:
                             The id of the pool_type to use as pool_type_id for
                             pool creation.
 
-                            Only pool-type-id 1 is supported
+                            Only pool-type-id 1 is supported.
```
[the remark this answers is not in the document]
Comment (barriebyron) [reply client/docs/swagger-ui/swagger.yaml:35]: thanks for noticing @kogisin 
- for title, the style is not to use punctuation (dot) at the end of the sentence
(our titles are probably longer than appropriate, I need to learn more)
- for description, we do need punctuation at the end of complete sentences

**Comment by barriebyron** [comment]
@kogisin can you take a look at the minor changes I just introduced, so make sure all is accurate and as expected?

**Review by kogisin — APPROVED** [review]
LGTM

**Pull request merged**